### DISCUSSION TITLE
How to instantiate a mil.nga.sf.geojson.MultiPolygon;MultiPolygon

### DISCUSSION THREAD

**Discussion by jorgerabello3** [ngageoint/simple-features-geojson-java#4] [Q&A] [answered]
Hello I'm using this lib in a personal project.

I need to do some unit tests and would like to know how I can instantiate an object of type mil.nga.sf.geojson.MultiPolygon;MultiPolygon.

I would like to do something like:

`mil.nga.sf.geojson.MultiPolygon;MultiPolygon multipolygon = new mil.nga.sf.geojson.MultiPolygon;MultiPolygon()`

Can you help me please ?

Thanks in advance

**Comment by bosborn** [comment, marked as answer]
`mil.nga.sf.geojson.MultiPolygon multiPolygon = new mil.nga.sf.geojson.MultiPolygon();`

or if you want to populate it with a core simple features MultiPolygon, something like...
```
mil.nga.sf.MultiPolygon multi = new mil.nga.sf.MultiPolygon();
mil.nga.sf.Polygon polygon = new mil.nga.sf.Polygon();
mil.nga.sf.LineString lineString = new mil.nga.sf.LineString();
lineString.addPoint(new Point(-122.358, 47.653));
lineString.addPoint(new Point(-122.348, 47.649));
lineString.addPoint(new Point(-122.348, 47.658));
lineString.addPoint(new Point(-122.358, 47.658));
lineString.addPoint(new Point(-122.358, 47.653));
polygon.addRing(lineString);
multi.addPolygon(polygon);
// Add additional polygons
mil.nga.sf.geojson.MultiPolygon multiPolygon = new mil.nga.sf.geojson.MultiPolygon(multi);
```

**Reply by jorgerabello3** [reply]
Wow thank u so much @bosborn  ! It will help me so much xD
